platform/android/java/lib/src/org/godotengine/godot/Godot.kt

```diff
@@ -914,6 +914,12 @@ class Godot(private val context: Context) {
 		}
 	}
 
+	@Keep
+	private fun getAccentColor(): Int {
+		val value = TypedValue()
+		context.theme.resolveAttribute(android.R.attr.colorAccent, value, true)
+		return value.data
```

Comment:
Since this value is unlikely to change during the app's lifecycle, let's use a kotlin lazy variable to retrieve it once when needed, then reuse the cached value.

Reply:
@m4gr3d The accent color can actually change if the user switches themes (i.e, between light and dark mode). Since this color is likely to be retrieved just once or infrequently, I don’t anticipate a significant impact from accessing it on demand.